text-utilities/text.css

```diff
@@ -0,0 +1,29 @@
+.h1{
+    font-size: 2rem;
+}
+.h2{
+    font-size: 1.75rem;
+}
+.h3{
+    font-size: 1.5rem;
+}
+.h4{
+    font-size: 1.25rem;
+}
+.h5{
+    font-size: 1rem;
+}
+.h6{
+    font-size: 0.875rem;
+}
```

Comment:
you can add padding and margin to it

Reply:
If I do that then it will create problem while using it any exiting code base project